frontend/src/components/Attendance/SendMyDailyStatus.jsx

```diff
@@ -3,62 +3,65 @@ import DatePicker from "react-datepicker";
 import "react-datepicker/dist/react-datepicker.css";
 import axios from "axios";
 import { useNavigate } from "react-router-dom";
+import Select from 'react-select';
 
 const SendMyDailyStatus = () => {
   const [startDate, setStartDate] = useState(new Date());
-  const [email, setEmail] = useState("");
+  const [email, setEmail] = useState([]);
   const [project_status, setProjectStatus] = useState("");
   const [status, setStatus] = useState("");
   const [working_hour, setWorkingHour] = useState("");
   const [task, setTask] = useState("");
-  const [data, setData] = useState([]);
-  const [filteredData, setFilteredData] = useState([]);
-  const [isLoading, setIsLoading] = useState(true); // Added loading state
+  const [emailData, setEmailData] = useState([]);
   const navigate = useNavigate();
 
-  const handleFilter = (value) => {
-    const res = data.filter((f) => f.email.toLowerCase().includes(value));
-    setFilteredData(res);
-  };
-
-  const handleStatusSubmit = async (e) => {
-    e.preventDefault();
-    try {
-      const response = await axios.post("http://localhost:4500/post_daily_status",{ withCredentials: true }, {
-        email,
-        date: startDate,
-        project_status,
-        status,
-        working_hour,
-        task,
-      });
-      const { success, message } = response.data;
-      if (success) {
-
-      } else {
-        console.log(message);
-      }
-    } catch (error) {
-      console.error("Login error", error);
-    }
-    navigate("/daily_status_updates");
-  };
-
   useEffect(() => {
     const getData = async () => {
+     
       try {
-        const response = await axios.get("http://localhost:4500/getEmail");
-        setData(response.data);
-        setFilteredData(response.data);
+        const response = await axios.get("http://localhost:4500/auth/getEmail");
+        setEmail(response.data.emails);
       } catch (error) {
         console.error("Error fetching data:", error);
-      } finally {
-        setIsLoading(false);
-      }
+      } 
     };
     getData();
   }, []);
 
+  const createEmailObjects = (emailArray) => (
+    emailArray.map(email => ({ value: email, label: email }))
+  );
+
+  const handleEmail = (selectedOptions) => {
+    const usersEmail = selectedOptions.map((e) => e.value);
+    setEmailData(usersEmail);
+  };
+
+  const handleStatusSubmit = async (e) => {
+    const token =localStorage.getItem("jwtToken");
+    e.preventDefault();
+    try {
+      await axios.post(
+        "http://localhost:4500/tasks",
```

Comment:
for http://localhost:4500/ create a separate constant in utils folder like export const BaseURL = http://localhost:4500/